lib/sprite.dart

```diff
@@ -23,7 +24,13 @@ class Sprite {
 
   Vector2 get originalSize => Vector2(_imageWidth, _imageHeight);
 
-  Vector2 get size => Vector2(src.width, src.height);
+  Vector2 get size => Vector2(bounds.width, bounds.height);
```

Comment:
this is the same as originalSize. maybe could also be called srcSize. this method is not necessary I think.

Reply:
Not necessarily, since the user can send in a size argument